pkg/template/template.go

```diff
@@ -666,11 +705,12 @@ func funcMap() template.FuncMap {
 		"pickContains": pickContains,
 		"pickSuffix":   pickSuffix,
 		"pickPrefix":   pickPrefix,
-		"toYaml":       toYaml,
+		"toYaml":       ToYaml,
```

Comment:
Why this change from camelCase? Rest seem to be following camelCase.

Reply:
This enables use of ToYaml function outside the package. This has been used to make the log pretty print.

Reply:
ok